modular_citadel/code/game/gamemodes/gangs/gang_items.dm

```diff
@@ -0,0 +1,410 @@
+
+/datum/gang_item/weapon/ammo/improvised_ammo
+	name = "Box of Buckshot"
+	id = "buckshot"
+	cost = 5
+	item_path = /obj/item/storage/box/lethalshot
+
+/datum/gang_item/weapon/pistol
+	name = "10mm Pistol"
+	id = "pistol"
+	cost = 25
+	item_path = /obj/item/gun/ballistic/automatic/pistol
+
+/datum/gang_item/weapon/ammo/pistol_ammo
+	name = "10mm Ammo"
+	id = "pistol_ammo"
+	cost = 10
+	item_path = /obj/item/ammo_box/magazine/m10mm
+
+/datum/gang_item/weapon/sniper
+	name = "Black Market .50cal Sniper Rifle"
+	id = "sniper"
+	cost = 35
+	item_path = /obj/item/gun/ballistic/automatic/sniper_rifle
+
+/datum/gang_item/weapon/ammo/sniper_ammo
+	name = "Smuggled .50cal Sniper Rounds"
+	id = "sniper_ammo"
+	cost = 15
+	item_path = /obj/item/ammo_box/magazine/sniper_rounds
+
+/datum/gang_item/weapon/ammo/sleeper_ammo
+	name = "Illicit Soporific Cartridges"
+	id = "sniper_ammo"
+	cost = 15
+	item_path = /obj/item/ammo_box/magazine/sniper_rounds/soporific
+
+/datum/gang_item/weapon/machinegun
+	name = "Mounted Machine Gun"
+	id = "MG"
+	cost = 45
+	item_path = /obj/machinery/manned_turret
+	spawn_msg = "<span class='notice'>The mounted machine gun features enhanced responsiveness. Hold down on the trigger while firing to control where you're shooting.</span>"
+
+/datum/gang_item/weapon/machinegun/spawn_item(mob/living/carbon/user, obj/item/device/gangtool/gangtool)
+	new item_path(user.loc)
+	to_chat(user, spawn_msg)
+
+/datum/gang_item/weapon/uzi
+	name = "Uzi SMG"
+	id = "uzi"
+	cost = 50
+	item_path = /obj/item/gun/ballistic/automatic/mini_uzi
+
+/datum/gang_item/weapon/ammo/uzi_ammo
+	name = "Uzi Ammo"
+	id = "uzi_ammo"
+	cost = 20
+	item_path = /obj/item/ammo_box/magazine/uzim9mm
+
+///////////////////
+//EQUIPMENT
+///////////////////
+
+/datum/gang_item/equipment
+	category = "Purchase Equipment:"
+
+
+/datum/gang_item/equipment/spraycan
+	name = "Territory Spraycan"
+	id = "spraycan"
+	cost = 1
+	item_path = /obj/item/toy/crayon/spraycan/gang
+
+/datum/gang_item/equipment/spraycan/spawn_item(mob/living/carbon/user, datum/team/gang/gang, obj/item/device/gangtool/gangtool)
+	var/obj/item/O = new item_path(user.loc, gang)
+	user.put_in_hands(O)
+
+/datum/gang_item/equipment/sharpener
+	name = "Sharpener"
+	id = "whetstone"
+	cost = 3
+	item_path = /obj/item/sharpener
+
+
+/datum/gang_item/equipment/emp
+	name = "EMP Grenade"
+	id = "EMP"
+	cost = 7
+	item_path = /obj/item/grenade/empgrenade
+
+/datum/gang_item/equipment/c4
+	name = "C4 Explosive"
+	id = "c4"
+	cost = 7
+	item_path = /obj/item/grenade/plastic/c4
+
+/datum/gang_item/equipment/frag
+	name = "Fragmentation Grenade"
+	id = "frag nade"
+	cost = 5
+	item_path = /obj/item/grenade/syndieminibomb/concussion/frag
+
+/datum/gang_item/equipment/stimpack
+	name = "Black Market Stimulants"
+	id = "stimpack"
+	cost = 12
+	item_path = /obj/item/reagent_containers/syringe/stimulants
+
+/datum/gang_item/equipment/implant_breaker
+	name = "Implant Breaker"
+	id = "implant_breaker"
+	cost = 10
```

Comment:
12 or 15 do to how easy it is to gang a sec officer or head of staff

Reply:
Gangs have no good way to stun/capture someone to actually use this, Its seen no prominent use so far. Even then its a option to keep someone alive instead of dead.